### ISSUE TITLE
questions about tot in GHMR_Loss

### ISSUE THREAD

**Issue by Kindpire** [libuyu/GHM_Detection#11]
Hi,
Thanks for your excellent work!
As you mentioned in #2, mask is used for add the samples to training or not. What I don't understand is, in retina_head.py, you used ghmr_loss as:
loss_reg = self.ghmr_loss.calc(
                bbox_pred,
                bbox_targets,
                bbox_weights) 
So bbox_weights is input as mask, however, as far as I understand, in anchor_target.py, I think bbox_weights is a weight matrix set positive sample to 1 and negative samples to 0.
bbox_weights = torch.zeros_like(anchors)
if len(pos_inds) > 0:
        bbox_weights[pos_inds, :] = 1.0
@libuyu Could you please explain this part with an example? Thanks!

**Comment by libuyu** [comment]
@Kindpire Your understanding is right. Since for the box regression task, only the positive boxes are valid, all the negative boxes and the boxes in the ignored regions are not used for training (because regressing a background box makes no sense).

**Comment by Kindpire** [comment]
Got it, thanks for the explanation!

**Comment by CaptainEven** [comment]
@libuyu Hi，how to compute label_weight in GHM_C loss?

**Comment by Wang-Wenqing** [comment]
> @libuyu Hi，how to compute label_weight in GHM_C loss?

Have you have answer about this question?
